### ISSUE TITLE
espanso package list not working

### ISSUE THREAD

**Issue by smeech** [federico-terzi/espanso#1860]
My command `espanso package list` produces the error:
```bash
unable to list packages: unable to list packages

Caused by:
    0: unable to parse package source file
    1: No such file or directory (os error 2)
```
I am able to `espanso package install` and `espanso package uninstall`, and `espanso package update` as long as I don't append `all` (which generates the second error).

The same problems apply in my Windows 10 VM, into which I copied my Espanso directories. The host is Linux Mint Xfce.

Any suggestions, short of reinstalling, as to how I can get this working?

**Comment by kpym** [comment]
It looks like one of your packages is the problem. You can try to update them one by one to find the one that is causing the problem.

Since you can't list them, you can go directly to the path obtained by `espanso path packages` and check the names.

**Comment by smeech** [comment]
Yes - I'll try that - after taking copies of the ones I've modified!

**Comment by smeech** [comment]
Thanks - that sorted it out.

**Comment by joh6nn** [comment]
@smeech I think it might be worth reopening this, with information about which package was causing issues and what the specific changes that affected it were, so that we can fix the `package list` command to not fail when it encounters that

**Comment by smeech** [comment]
It's likely the problem arose because, in my initial enthusiasm on discovering Espanso, rather than reading the docs, I downloaded some raw packages files from the hub repo, rather than use the `espanso package install` command. Consequently, they didn't get added to whatever index the `espanso package` commands use.
Once I followed your advice and recreated a clean slate the commands work fine.
I suppose one might ask that Espanso parse the actual packages on the system, but there's no guarantee they match the hub, just because they share a name.
The Issue will remain here for anyone who encounters the same problem and comes looking for a solution.
